src/components/Doc/ComponentDoc.js

```diff
@@ -19,6 +19,7 @@ export const ComponentDoc = ({
   name,
   pageTitle,
   designSystemLink,
+  stable,
```

Comment:
When we're ready, swapping this out with a prop name like `version` or `grommetVersion` could provide flexibility to promote new components in the current release and swap out messaging. But agree with your [comment](https://github.com/grommet/grommet-site/pull/416#issuecomment-1242353846) that we can treat as a future enhancement. Just planting seeds.

Reply:
Good note
Issue created: https://github.com/grommet/grommet-site/issues/417